src/flatConfigEditor.ts

```diff
@@ -422,12 +422,8 @@ export class FlatConfigEditor implements vscode.CustomTextEditorProvider {
     return new Promise(async (resolve, reject) => {
       try {
         const gitClient = new VSCodeGit()
-        await gitClient.activateExtension()
-        await gitClient.waitForRepo(3)
-
-        // Next, let's grab the repo name.
+        await gitClient.waitForRepo()
```

Comment:
We only ever call `waitForRepo` here... As such, I removed the "# of times" argument from the function and co-located it in the function implementation itself.